molgenis-data-excel/src/main/java/org/molgenis/data/excel/ExcelEntity.java

```diff
@@ -23,6 +24,8 @@
 	private final Map<String, Integer> colNamesMap;
 	private final List<CellProcessor> cellProcessors;
 
+	private transient Map<String, Object> cachedValueMap;
```

Comment:
Why are not all class parameters are final transient?
cachedValueMap can this not be already instantiated?